code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -214,6 +214,20 @@
 		M.knocked_out = max(M.knocked_out, 20)
 		M.drowsyness = max(M.drowsyness, 30)
 
+/datum/reagent/mothpheromones
+	name = "Moth pheromones"
+	id = "mothpheromones"
+	description = "Basicly the liquid concept of fluttering towards a lamp."
+	reagent_state = LIQUID
+	color = "#8fddf7" // rgb: 143, 221, 247
+
+/datum/reagent/mothpheromones/on_mob_life(mob/living/M)
+	. = ..()
+	if(!.) return
+	if(has_species(M, "Moth"))
+		if(prob(7)) M.emote(pick("twitch","drool","moan","gasp"))
```

Comment:
Same line control statement.

Reply:
Also, an alien tag "IS_MOTH" has been already made, you may want to use it instead.
`if(alien == IS_MOTH)`, should be it.